package.bat

```diff
@@ -0,0 +1,13 @@
+echo Please ensure PYTHON is set to the directory of Python installation at version 3.6.
+set OLD_PATH=%PATH%
+PATH=%PYTHON%;%PYTHON%\Scripts;%PATH%
+python --version
+pip --version
+pip install cx_Freeze
```

Comment:
I don't think it should be in `requirements.txt`; it's not required when running Wangcheck as a script (eg. when building from source), only when packaging it.